dmaws/rds.py

```diff
@@ -287,59 +294,28 @@ def clean_database_for_staging(self):
         self.log("Delete audit events")
         self.cursor.execute("DELETE FROM audit_events")
 
+        self.log("Delete draft framework agreements")
```

Comment:
Should we drop all of them? I don't think there's anything special about the draft ones. And the files aren't there anyway, so it's not like they'd be useful for the admin flow.